scarb/src/core/registry/client/local.rs

```diff
@@ -213,6 +213,7 @@ fn edit_records(records_path: &Path, func: impl FnOnce(&mut IndexRecords)) -> Re
         .read(true)
         .write(true)
         .create(true)
+        .truncate(false)
```

Comment:
Why?

Reply:
```
warning: file opened with `create`, but `truncate` behavior not defined
   --> scarb/src/core/registry/client/local.rs:215:10
    |
215 |         .create(true)
    |          ^^^^^^^^^^^^- help: add: `.truncate(true)`
    |
    = help: if you intend to overwrite an existing file entirely, call `.truncate(true)`
    = help: if you instead know that you may want to keep some parts of the old file, call `.truncate(false)`
    = help: alternatively, use `.append(true)` to append to the file instead of overwriting it
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#suspicious_open_options
    = note: `#[warn(clippy::suspicious_open_options)]` on by default
```